localenv/mock-account-servicing-entity/app/entry.server.tsx

```diff
@@ -39,6 +39,11 @@ if (!global.__seeded) {
   })
     .then(() => {
       global.__seeded = true
+      setTimeout(() => {
+        console.log('Server started with the following environment variables:')
+        console.log(`OPERATOR_TENANT_ID: ${process.env.OPERATOR_TENANT_ID}`)
+        console.log(`SIGNATURE_SECRET: ${process.env.SIGNATURE_SECRET}`)
+      }, 2000)
```

Comment:
You have to submit these in the admin ui to use it now. so im logging them here to streamline that (dont have to hunt them down in the docker-compose). Could not for the life of me figure out how to get them to log after the seed script without the timeout.